packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -3698,7 +3698,7 @@ export function pushStartInstance(
       return pushSelfClosing(target, props, type);
     }
     // These are reserved SVG and MathML elements, that are never custom elements.
-    // https://w3c.github.io/webcomponents/spec/custom/#custom-elements-core-concepts
+    // https://www.w3.org/TR/custom-elements/
```

Comment:
I think this is the right link: https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-core-concepts

Same below

Reply:
Thanks for the feedback! It has been updated with new link.